zerver/openapi/zulip.yaml

```diff
@@ -8984,6 +8984,15 @@ paths:
                           Whether the server needs upgrade.
 
                           **Changes**: New in Zulip 5.0 (feature level 74).
+                      event_queue_longpoll_timeout_seconds:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          Client-side polling timeout for get_events, in seconds.
```

Comment:
Let's say "Recommended client-side HTTP request timeout for [GET /events](/api/get-events) calls.  This is guaranteed to be somewhat greater than the heartbeat frequency.  It is important that clients respect this parameter, so that increases in the heartbeat frequency do not break clients."

And then we should (1) add text in the `GET /events` documentation mentioning this and (2) open issues with zulip-mobile, zulip-terminal, zulip-js, and python-zulip-api to respect this parameter if present, defaulting to 90s if not present.